src/main/java/com/epam/university/java/core/task019/RobotImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.university.java.core.task019;
+
+/**
+ * Implementation class for Robot.
+ *
+ * @author Sergei Titov
+ */
+public class RobotImpl implements Robot {
+
+    // default orientation is "to the right"
+    private int deltaX = 1;
+    private int deltaY = 0;
+
+    RobotPositionImpl position = new RobotPositionImpl();
```

Comment:
It's better to declare properties as private members

Reply:
indeed